install_config/configuring_authentication.adoc

```diff
@@ -2105,7 +2105,15 @@ by `oc version`)
 the user agent will be:
 
 ----
-oc/v3.3.0 (linux/amd64) openshift/f034127
+oc/v3.3.0+f034127 (linux/amd64) openshift/f034127
```

Comment:
Should be 'kubernetes' version , not the oc version, for example:
[root@dhcp-140-138 ~]# oc version 
oc v3.11.207
kubernetes v1.11.0+d4cacc0
features: Basic-Auth GSSAPI Kerberos SPNEGO


`oc get project -v 7` show:
User-Agent: oc/v1.11.0+d4cacc0 (linux/amd64) kubernetes/d4cacc0